src/Core/Settings.h

```diff
@@ -800,6 +800,7 @@ static constexpr UInt64 operator""_GiB(unsigned long long value)
     M(Int64, record_consume_timeout_ms, 100, "Timeout of consuming record", 0) \
     M(Int64, kafka_fetch_wait_max_ms, 100, "When reading from Kafka, max wait time", 0) \
     M(Int64, kafka_fetch_max_bytes, 52428800, "When reading from Kafka, max bytes to fetch per read", 0) \
+    M(Int64, kafka_max_message_size, 1024 * 1024, "When writing to Kafka, the max size of a message in bytes", 0) \
```

Comment:
i was thinking if we have general way doing this like the existing `max_ingest_block_size` or `max_block_size`